src/test/java/com/desiato/puresynth/models/UserControllerTest.java

```diff
@@ -67,6 +64,42 @@ public void deleteUser_ShouldDeleteUserAndReturnOk() throws Exception {
                 .andReturn();
     }
 
+    @Test
```

Comment:
It would be also good to add some error scenario test, that you return e.g. 404.

Reply:
yes, good idea!